Code/GraphMol/FMCS/Wrap/rdFMCS.cpp

```diff
@@ -110,6 +119,9 @@ struct mcsresult_wrapper {
                       "number of atoms in MCS")
         .def_readonly("numBonds", &RDKit::MCSResult::NumBonds,
                       "number of bonds in MCS")
+        .def("queryMol", RDKit::getQueryMol,
```

Comment:
If we need to use the function, it should probably be named `GetQueryMol()` since it's different from the other attributes.
Can't we just use the `QueryMol` attribute directly though?

Reply:
@greglandrum Ditto.